charts/newrelic-infra-operator/templates/deployment.yaml

```diff
@@ -19,9 +19,9 @@ spec:
         {{- toYaml .Values.podAnnotations | nindent 8 }}
         {{- end }}
       labels:
-        {{- include "newrelic-infra-operator.labels" . | nindent 8 }}
+        {{- include "newrelic.common.labels" . | nindent 8 }}
     spec:
-      serviceAccountName: {{ template "newrelic-infra-operator.serviceAccountName" . }}
+      serviceAccountName: {{ template "newrelic.common.serviceAccount.name" . }}
       {{- if .Values.podSecurityContext.enabled }}
       securityContext:
         runAsUser: {{ .Values.podSecurityContext.runAsUser }}
```

Comment:
Use the common-library `securityContext` helper. Seems that this would be a breaking change because we no longer support the `runAsUser` config, so you should add a compatibility layer. You can take a look for instance at this PR https://github.com/newrelic/nri-kube-events/pull/108.

This can be addressed in a follow up PR.